server/src/acccount-manager/dto/create-user.dto.ts

```diff
@@ -1,36 +1,48 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
 import { IsEmail, IsNotEmpty, IsBoolean, IsOptional, Validate } from 'class-validator';
 import { InterestsProps, InterestNamesIsArray } from '../../shared/interests.validator';
 import { Interests } from '../../shared/interests.dto';
 
 export class CreateUserDto {
+  @ApiProperty()

```

Comment:
In this dto file, we can remove all the `@ApiProperty()` annotations from `@nestjs/swagger`. The OpenApi plugin will pick up the required/optional properties as well as the interest type based on the typescript types.